examples/seismic/source.py

```diff
@@ -266,3 +266,34 @@ def wavelet(self, f0, t):
         tcut = 1.5 / agauss
         s = (t-tcut) * agauss
         return np.exp(-2*s**2) * np.cos(2 * np.pi * s)
+
+
+class DGaussSource(WaveletSource):
+    """
+    Symbolic object that encapsulate a set of sources with a
+    pre-defined 1st derivative wavelet of a Gaussian Source:
+
+    Note: For visualizing the second or third order derivative of Gaussian wavelets,
```

Comment:
numpydoc

Reply:
meaning, this should go under 
```
Notes
------
...
```

which in turn should go below the ```Parameters ``` etc etc

Reply:
OK